packages/clerk-js/src/ui/foundations/colors.ts

```diff
@@ -37,23 +37,19 @@ export const neutralAlpha = Object.freeze({
 } as const);
 
 export const colors = Object.freeze({
-  // Colors that are not affected by `alphaShadesMode`
   avatarBorder: neutralAlpha.neutralAlpha200,
   avatarBackground: neutralAlpha.neutralAlpha400,
   modalBackdrop: neutralAlpha.neutralAlpha700,
-  activeDeviceBackground: whiteAlpha.whiteAlpha200,
   // Themable colors
   ...neutralAlpha,
   ...whiteAlpha,
   colorBackground: 'white',
   colorInputBackground: 'white',
-  colorText: '#131316',
+  colorText: '#212126',
```

Comment:
Why are we changing this?

Reply:
That has been discussed with the design team and we concluded to use this color.